crossplane-argocd-gitops/envs/dev/values.yaml

```diff
@@ -0,0 +1,17 @@
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: https://github.com/aws-samples/eks-blueprints-workloads
+    targetRevision: main
+clusterA:
+  clusterName: 
+clusterB:
+  clusterName: 
+common:
+  providerConfigAWSName:  
+  eksConnectorRoleName:  
+  accountId:  
```

Comment:
Please clarify why we are passing account here? This is deployed in AWS and has an IRSA role, do we need to pass this.

Same question about region? How are we handling the case when deploying to multiple regions from the same mgmt cluster?

Reply:
we need accounId since  the IRSA role of eks provider pod in mngmt cluster needs to assume platform role of workload clusters as follows in common-provider-config-aws.yaml
  assumeRoleChain:
    - roleARN: "arn:aws:iam::{{ .Values.common.accountId }}:role/{{ .Values.common.eksConnectorRoleName }}"

For now, I kept region common for all clusters and kept it under common section in values.yaml. this can be easily moved later if needed to respective cluster sections i.e. clusterA and clusterB in values.yaml